src/app/components/login/login.component.scss

```diff
@@ -100,3 +100,12 @@ input {
   color: $gray-color;
   font-size: $gray-font-size;
 }
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus textarea:-webkit-autofill,
+textarea:-webkit-autofill:hover textarea:-webkit-autofill:focus,
+select:-webkit-autofill,
+select:-webkit-autofill:hover,
+select:-webkit-autofill:focus {
+  -webkit-box-shadow: 0 0 0px 1000px #ffffff inset !important;
```

Comment:
Avoid `!important` as much as possible. Only use it if there really is no other way.